enigmatry-entry-blueprint-app/src/environments/environment.prod.ts

```diff
@@ -6,5 +6,8 @@ export const environment = {
     authority: '__azureAdAuthority__',
     scopes: ['__azureAdScopes__']
   },
+  applicationInsights: {
+    connectionString: '__applicationInsightsConnectionString__'
```

Comment:
Check if this is added to parameters.xml and deploy profile for the app. Blueprint template is not deployed to test, so values in the deploy profile do not matter, you can use same ones as for development.